app/models/drug.rb

```diff
@@ -0,0 +1,37 @@
+class Drug < ActiveYaml::Base
+  set_root_path "config/data"
+  set_filename "drugs"
+
+  CREATED_TIME ||= File.ctime("config/data/drugs.yml").in_time_zone("UTC")
+  UPDATED_TIME ||= File.mtime("config/data/drugs.yml").in_time_zone("UTC")
```

Comment:
Sets the drugs' updated at to whenever the file was last updated. This will resync all drugs whenever the file is modified, but that should be okay since its not a large list.